File: src/ss5/baitap1/Circle.java
package ss5.baitap1;
public class Circle {
    // Thuộc tính private
    private double radius = 1.0;
    private String color = "red";

    // Hàm tạo không tham số
    public Circle() {
    }

    // Hàm tạo có tham số radius
    public Circle(double radius) {
        this.radius = radius;
    }

    // Phương thức private: getRadius
    private double getRadius() {
        return radius;
    }

    // Phương thức private: getArea
    private double getArea() {
        return Math.PI * radius * radius;
    }

    // Phương thức public mới để in thông tin
    public void printDetails() {
        System.out.println("Radius: " + getRadius());
        System.out.println("Area: " + getArea());
    }
}

File: src/ss5/baitap1/Circle.kt
package ss5.baitap1

class Circle {
    // Phương thức private: getRadius
    // Thuộc tính private
    private var radius = 1.0
    private val color = "red"

    // Hàm tạo không tham số
    constructor()

    // Hàm tạo có tham số radius
    constructor(radius: Double) {
        this.radius = radius
    }

    private val area: Double
        // Phương thức private: getArea
        get() = Math.PI * radius * radius

    // Phương thức public mới để in thông tin
    fun printDetails() {
        println("Radius: " + radius)
        println("Area: " + area)
    }
}
